sympy/integrals/tests/test_integrals.py

```diff
@@ -1070,17 +1070,36 @@ def test_issue_2708():
 
 def test_issue_8368():
     assert integrate(exp(-s*x)*cosh(x), (x, 0, oo)) == \
-        Piecewise((pi*Piecewise((-s/(pi*(-s**2 + 1)), Abs(s**2) < 1),
-        (1/(pi*s*(1 - 1/s**2)), Abs(s**(-2)) < 1), (meijerg(((S(1)/2,), (0, 0)),
-        ((0, S(1)/2), (0,)), polar_lift(s)**2), True)),
-        And(Abs(periodic_argument(polar_lift(s)**2, oo)) < pi,
-        cos(Abs(periodic_argument(polar_lift(s)**2, oo))/2)*sqrt(Abs(s**2)) -
-        1 > 0, Ne(s**2, 1))), (Integral(exp(-s*x)*cosh(x), (x, 0, oo)), True))
+        Piecewise(
+            (   pi*Piecewise(
```

Comment:
The formatting issue still stands. Perhaps just dedenting all the text one level back (or the parens one level forward) would make it more readable, and still not have spaces in parens? Something like:

```
(sub expression,
    (subexpression1,
     subexpression2),
 otherstuff)...
```
idk. I'd be ok with whatever seems reable to you, but has *no spaces* between the parens and the actual expression.